python/ml4ir/base/features/feature_layer.py

```diff
@@ -1,14 +1,12 @@
-# flake8: noqa
```

Comment:
Fixed code complexity and consolidated the feature layer methods into one single method that handles both SequenceExamples and Examples